python/test/unit/language/test_core.py

```diff
@@ -1467,6 +1467,29 @@ def kernel(X, SHAPE0: tl.constexpr, SHAPE1: tl.constexpr):
     assert torch.min(x).item() == 0.0
 
 
+@pytest.mark.interpreter
```

Comment:
if the goal is to test a specific behavior in a pass we should write a minimized lit test.